common-api/src/main/java/org/pmiops/workbench/SqlDialect/CommonTestDialect.java

```diff
@@ -0,0 +1,18 @@
+package org.pmiops.workbench.sqldialect;
+
+import org.hibernate.dialect.MySQLDialect;
+import org.hibernate.dialect.function.SQLFunctionTemplate;
+import org.hibernate.type.StandardBasicTypes;
+
+public class CommonTestDialect extends MySQLDialect {
```

Comment:
Nice work on reusing this!

Please change the name of the directory that contains this to "sqldialect"... there shouldn't be any capital letters in package names.

I also don't know if we need a separate "sqldialect" package; this could just go in cdr probably.

Reply:
I changed the package name with the lowercase. The change was not reflected somehow.
Anyway moved the file to cdr.